taipy/core/config/core_section.py

```diff
@@ -28,9 +28,12 @@ class CoreSection(UniqueSection):
 
     Attributes:
         root_folder (str): Path of the base folder for the taipy application. The default value is "./taipy/"
-        storage_folder (str): Folder name used to store Taipy data. The default value is ".data/". It is used in
+        storage_folder (str): Folder name used to store user data. The default value is ".user_data/". It is used in
             conjunction with the *root_folder* attribute. That means the storage path is <root_folder><storage_folder>
-            (The default path is "./taipy/.data/").
+            (The default path is "./taipy/.user_data/").
+        tp_storage_folder (str): Folder name used to store Taipy data. The default value is ".taipy_data/". It is used
+            in conjunction with the *root_folder* attribute. That means the storage path is
+            <root_folder><storage_folder> (The default path is "./taipy/.taipy_data/").
```

Comment:
Are we sure about this? I don't think the `.taipy` folder ever exists

Reply:
Im not sure. I kept the current text, just replaced .data with the new folder name. Maybe @jrobinAV can verify if this docstring is correct.

Reply:
`<storage_folder>` should be renamed by `taipy_storage_folder` (or whatever the name of the attribute we chose).
```suggestion
            <root_folder><taipy_storage_folder> (The default path is "./taipy/.taipy/").
```